.github/workflows/acctest.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
           export PATH=$PATH:$(pwd)/bin
           wget https://releases.hashicorp.com/terraform/0.12.17/terraform_0.12.17_linux_amd64.zip
           unzip terraform_0.12.17_linux_amd64.zip && sudo mv terraform /usr/local/bin
-          docker run -d -p 6650:6650 -p 8080:8080 -v $PWD/data:/pulsar/data apachepulsar/pulsar:2.4.1 bin/pulsar standalone
+          docker run -d -p 6650:6650 -p 8080:8080 -v $PWD/data:/pulsar/data apachepulsar/pulsar:2.7.0 bin/pulsar standalone
```

Comment:
Note: This change from Pulsar 2.4.1 to 2.7.0 was intended to fix these integration test errors

```
=== RUN   TestNamespace
    testing.go:635: Step 0 error: errors during apply:
        
        Error: ERROR_CREATE_NAMESPACE_CONFIG: ERROR_UPDATE_NAMESPACE_CONFIG: 2 errors occurred:
        	* GrantNamespacePermission: code: 501 reason: Authorization is not enabled
        	* GrantNamespacePermission: code: 501 reason: Authorization is not enabled
        
        
        
          on /var/folders/f2/7c9_sq9j46sb4nbywp3w1dbxr8xj5b/T/tf-test019164888/main.tf line 22:
          (source code not available)
```